Makefile

```diff
@@ -1,9 +1,43 @@
 TOOL_VERSION = 2.1.12
 TEST_DATA = test/simpleTestForGenerator
-
+GIT_AUTHOR = License Publisher (maintained by Gary O'Neall) <gary@sourceauditor.com>
+LICENSE_DATA_REPO_NO_SCHEME = github.com/spdx/license-list-data.git
+LICENSE_DATA_REPO = https://$(LICENSE_DATA_REPO_NO_SCHEME)
+LICENSE_DATA_URL = https://$(GITHUB_TOKEN)@$(LICENSE_DATA_REPO_NO_SCHEME)
+LICENSE_OUTPUT_DIR = .tmp
+VERSION = $(shell git describe --always || echo 'UNKNOWN')
+RELEASE_DATE = $(shell date '+%Y-%m-%d')
+COMMIT_MSG = License list build $(VERSION) using license list publisher $(TOOL_VERSION)
+RELEASE_MSG = Adding release matching the license list XML tag $(VERSION)
+#TODO Change the license data repo to license-list-xml before merging pull request
+	
 .PHONY: validate-canonical-match
-validate-canonical-match: licenseListPublisher-$(TOOL_VERSION).jar-valid resources/licenses-full.json $(TEST_DATA) .tmp
-	java -jar -DLocalFsfFreeJson=true -DlistedLicenseSchema="schema/ListedLicense.xsd" licenseListPublisher-$(TOOL_VERSION).jar LicenseRDFAGenerator src .tmp 1.0 2000-01-01 $(TEST_DATA) expected-warnings
+validate-canonical-match: licenseListPublisher-$(TOOL_VERSION).jar-valid $(TEST_DATA) $(LICENSE_OUTPUT_DIR)
+	java -jar -DLocalFsfFreeJson=false -DlistedLicenseSchema="schema/ListedLicense.xsd" licenseListPublisher-$(TOOL_VERSION).jar LicenseRDFAGenerator src $(LICENSE_OUTPUT_DIR) 1.0 2000-01-01 $(TEST_DATA) expected-warnings
+
+.PHONY: deploy-license-data
+deploy-license-data: licenseListPublisher-$(TOOL_VERSION).jar-valid $(TEST_DATA)
+	rm -rf $(LICENSE_OUTPUT_DIR)
+	git clone $(LICENSE_DATA_URL) $(LICENSE_OUTPUT_DIR) --quiet --depth 1
+	# Clean out the old data directories
+	find "$(LICENSE_OUTPUT_DIR)" -mindepth 1 -maxdepth 1 -name .git -prune -o -type d -exec rm -rf {} \+
+	rm -f $(LICENSE_OUTPUT_DIR)/licenses.md
+	java -jar -DLocalFsfFreeJson=false -DlistedLicenseSchema="schema/ListedLicense.xsd" licenseListPublisher-$(TOOL_VERSION).jar LicenseRDFAGenerator src $(LICENSE_OUTPUT_DIR) $(VERSION) $(RELEASE_DATE) $(TEST_DATA) expected-warnings
+	
+	git -C "$(LICENSE_OUTPUT_DIR)" add -A .
+	git -C "$(LICENSE_OUTPUT_DIR)" commit --author "$(GIT_AUTHOR)" -m "$(COMMIT_MSG)"
+	git -C "$(LICENSE_OUTPUT_DIR)" push origin
+	
+.PHONY: release-license-data
+release-license-data: deploy-license-data
+	if [[ $VERSION =~ .+-g[a-f0-9]{7} ]]
```

Comment:
You'll need semicolons here (and `\` line continuation unless you collapse the whole `if` block onto one line).  Otherwise `make` splits on the newlines and tries to execute them separately:

```console
$ cat Makefile 
test:
        if [ 1 = 1 ]
        then
                echo true
        else
                echo false
        fi
$ make
if [ 1 = 1 ]
/bin/sh: -c: line 1: syntax error: unexpected end of file
make: *** [Makefile:2: test] Error 1
```